### DISCUSSION TITLE
Is there any plan to release a new version of clikt with the mordant library 2.4.0 ?

### DISCUSSION THREAD

**Discussion by CharlyRien** [ajalt/clikt#502] [General]
The question is in the title. :)
I would love to switch from a third-party progressbar library to directly the one embedded in Clikt to remove some dependencies in my project. But I am waiting for the progressbar improvements done in 2.4.0. 

Do you plan to release something related to that? 

Thanks for your work in any case. :)

**Comment by ajalt** [comment]
You don't need to wait for a new clikt release, you can add an explicit dependency on the mordant version you want `implementation("com.github.ajalt.mordant:mordant:2.4.0")`

**Reply by CharlyRien** [reply]
I was afraid that would break clikt.  Thanks for the workaround.

**Reply by ajalt** [reply]
Any release other than a major version bump (2.x -> 3.x) should be binary compatible.
